src/components/Tipseen/__stories__/tipseen.stories.mdx

```diff
@@ -0,0 +1,228 @@
+    <Tipseen 
+      position={Tipseen.positions.RIGHT} 
+      modifiers={modifiers}
+      content={
+        <TipseenContent
+          isDismissHidden={false}
+          title="This is a title"
+          isDismissHidden
+          children="Popover message will appear here loremipsum dolor samet… "
+        />
+      }
+    >
+      <div className="monday-style-story-tipseen_container" />
+    </Tipseen>
+  </Story>
+</Canvas>
+
+### Tipseen with a wizard
+Use when the tip is too long or when you want to teach something in steps.
+<Canvas>
+  <Story name="Tipseen with a wizard">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <Tipseen
+          modifiers={modifiers}
+          position={Tipseen.positions.RIGHT} 
+          content={
+            <TipseenWizard 
+              title="This is a title"
+              steps={content}
+              activeStepIndex={2}
+            />
+          }
+        >
+          <div className="monday-style-story-tipseen_container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+### Tipseen with image
+<Canvas>
+  <Story name="Tipseen with image">
+    {() => {
+      const content = [
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>,
+        <div>Popover message will appear here loremipsum dolor samet…</div>
+      ];
+      return (
+        <Tipseen 
+          position={Tipseen.positions.RIGHT}
+          modifiers={modifiers}
+          isCloseButtonOnImage
+          content={
+            <>
+              <TipseenImage
+                className="monday-style-story-tipseen_image"
+                src={picture}
+              />
+              <TipseenWizard 
+                title="This is a title"
+                steps={content}
+                activeStepIndex={2}
+              />
+            </>
+          }
+        >
+          <div className="monday-style-story-tipseen_big-container" />
+        </Tipseen>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+        <>
+          <div>
+            <div className="monday-style-story-tipseen_column-item">
+              <Icon icon={Board} iconSize="24" /><span>💫 Research tasks</span>
+            </div>
+            <Tipseen
+              modifiers={modifiers}
```

Comment:
please add this comment above the modifiers prop:
"The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards. Therefore, there is no need to move this prop in your implementations."

Reply:
please add comment written above